.github/workflows/pr-verify.yml

```diff
@@ -0,0 +1,24 @@
+name: PR Verify
```

Comment:
This will add safety to the template repository to ensure we aren't introducing changes that will cause this to fail in an out of the box app. I can remove this as well or move it to circle CI